perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1032,6 +1032,21 @@ public Member setStatus(PerunSession sess, Member member, Status status, String
 		return getMembersManagerBl().setStatus(sess, member, status, message);
 	}
 
+	@Override
+	public void suspendMemberTo(PerunSession sess, Member member, Date suspendedTo) throws InternalErrorException, MemberNotExistsException, PrivilegeException {
+		Utils.checkPerunSession(sess);
+		Utils.notNull(suspendedTo, "suspendedTo");
```

Comment:
There is no way to "clear" suspended state otherwise than setting date in past. We should either allow null and properly clear db column or add method to un-suspend member.

Reply:
Method "unsuspendMember" was added. Good point!